src/videojs.ima.js

```diff
@@ -491,6 +502,28 @@
     }.bind(this);
 
     /**
+     * Syncs controls when an ad pauses.
+     * @param {google.ima.AdEvent} adEvent The AdEvent thrown by the AdsManager.
+     * @private
+     */
+    var onAdPaused_ = function(adEvent) {
+      showPauseButton();
```

Comment:
Don't you want to show the play button here, and pause button when the ad is resuming?

Reply:
Method names are bad and I feel bad - they will be fixed in the future. onAdPaused should really be showPause and onAdResumed should be showPlay